packages/shared/lib/services/nango-config.service.ts

```diff
@@ -576,7 +576,7 @@ function buildActions({
             version: action.version || null,
             last_deployed: action.updated_at || null,
             attributes: action.attributes || {},
-            returns: action.output as string[],
+            returns: Array.isArray(action.output) ? action.output : action.output ? [action.output] : [],
```

Comment:
not sure it is more readable but `[].concat(action.output)` should be equivalent

Reply:
typescript does not like that since action.output can be string or undefined